planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/path.cpp

```diff
@@ -614,6 +614,17 @@ std::vector<lane_change::TrajectoryGroup> generate_frenet_candidates(
            calc_average_curvature(p2.lane_changing.curvatures);
   });
 
+  if (trajectory_groups.size() > 1) {
+    auto subrange = ranges::subrange(trajectory_groups.begin() + 1, trajectory_groups.end());
+
+    auto remove_start = ranges::remove_if(subrange, [&](const auto & traj) {
+      return calc_average_curvature(traj.lane_changing.curvatures) >
```

Comment:
*Average* curvature is calculated, but the threshold is for the *max* curvature.
To me checking the max is better, but it may cause issues if the curvature is very noisy.